idea/src/main/kotlin/io/paddle/idea/settings/global/PaddleRegistryInteract.kt

```diff
@@ -0,0 +1,45 @@
+package io.paddle.idea.settings.global
+
+import io.paddle.utils.config.ConfigurationYAML
+import org.yaml.snakeyaml.DumperOptions
+import org.yaml.snakeyaml.Yaml
+import java.nio.file.Path
+import kotlin.io.path.readText
+
+object PaddleRegistryInteract {
+
+    private val PADDLE_HOME: Path
```

Comment:
That's hard to use `Locations` because that's an extension of the project

Reply:
Aha, got it. I guess this is a design mistake, made by myself. As in IDEA, there should be project-wide extensions and application-wide. Anyway, I don't really want to have things like PADDLE_HOME in several places, could you please move this `PaddleRegistryInteract` to `:core` submodule, call it somewhat like `PaddleApplicationSettings`, and make the `Locations` extension delegate to this class? So that PADDLE_HOME (etc.) would be declared only once in the code!